app/views/summary/index.html.haml

```diff
@@ -0,0 +1,43 @@
+%div.inset
+  %table.table.table-striped.table-bordered.table-responsive
+    %caption
+      %span.namespace= @summary.client_namespace
+      Requests: Fiscal Year
+      = @summary.fiscal_year
```

Comment:
Sorry, I wasn't clear. I wanted to understand the performance implications of the design I had chosen, to see if summarizing via Ruby vs summarizing via SQL was fast enough to give a reasonable user experience. Since the current fiscal year (2016) only has a couple months' worth of data, I wanted to be able to summarize an entire year for comparison. Ergo, needed to be able to easily select a year via the URL, defaulting to the current year.

So I am using it during development, and since it provides helpful historical comparison, year-over-year, I would like to keep it.

This whole summary feature is intentionally undocumented and under-scoped till we can get some user feedback. I've put out a call for that feedback so we can move forward.